app/controllers/application_controller.rb

```diff
@@ -8,6 +8,8 @@ class ApplicationController < ActionController::Base
 
   def configure_permitted_parameters
     devise_parameter_sanitizer.for(:sign_up) << :username
+    devise_parameter_sanitizer.for(:sign_up) << :avatar
```

Comment:
you can add them as an array `devise_parameter_sanitizer.for(:sign_up) << [:username, :avatar]`